railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -162,7 +162,9 @@ def master_key
       require_relative "../master_key/master_key_generator"
 
       after_bundle do
-        Rails::Generators::MasterKeyGenerator.new.add_master_key_file
+        key_generator = Rails::Generators::MasterKeyGenerator.new
```

Comment:
With this change I don't think we're ever calling `add_master_key_file` without also calling `ignore_master_key_file`. Let's just make sure the `add_master_key_file` calls `ignore_master_key_file` internally as well.